embedchain/pipeline.py

```diff
@@ -312,6 +312,15 @@ def _mark_data_as_uploaded(self, data_hash):
         )
         self.connection.commit()
 
+    def get_data_sources_by_app_id(self, app_id):
```

Comment:
You don't need to pass the app_id here, right? You can just access it by doing `self.local_id`

Reply:
Absolutely, I missed it. Thanks!